shared/common-adapters/avatar.js.flow

```diff
@@ -12,7 +12,9 @@ type CommonProps = {
   loadingColor?: string,
   forceLoading?: boolean,
   following?: ?boolean,
-  followsYou?: ?boolean
+  followsYou?: ?boolean,
+  hasBackgroundColor?: string,
```

Comment:
The name kinda sounds like the prop is a boolean value (as in it asks a yes/no question "Does this have background color"). I think simply `backgroundColor` would be more clear that a hex-color/string is expected w/o requiring the API consumer to read the implementation.